src/Sentry/Laravel/Tracing/EventHandler.php

```diff
@@ -182,14 +182,22 @@ protected function queryExecutedHandler(DatabaseEvents\QueryExecuted $query): vo
         $context = new SpanContext();
         $context->setOp('db.sql.query');
         $context->setDescription($query->sql);
+        $context->setData([
+            'db.name' => $query->connection->getDatabaseName(),
+            'db.system' => $query->connection->getDriverName(),
+            // 'server.address' => ,
+            // 'server.port'=> ,
```

Comment:
@stayallive any idea how we can elegantly fetch these values?

We would also need to distinguish between TCP and socket connections, `server.address` vs `server.socket.address`.

Reply:
Does it need to be elegant though... I think we can probably coorce the driver to give us those details. Is there also a way to say it's a file? For sqlite for example?

Reply:
If it works, it works 🪨😬

@AbhiPrasad about the sqlite case

Reply:
We can forgo `server.address` if it's just a file IMO